packages/core-components/src/components/SelectElement/index.vue

```diff
@@ -100,7 +100,7 @@ const TOGGLE_BUTTON = {
 
 interface Props {
   element?: ContentElement | null;
-  allowedTypes: string[];
+  allowedTypes: Array<{ type: string; schemaConfig: any }>;
```

Comment:
schemaConfig might be misleading in this context. Consider renaming it to contentElementSchemaConfig